checks.d/postgres.py

```diff
@@ -177,7 +177,7 @@ class PostgreSql(AgentCheck):
             ('schemaname', 'schema')
         ],
         'metrics': {
-            'pg_stat_user_tables': ('postgresql.total_tables', GAUGE),
+            'pg_stat_user_tables': ('postgresql.table.count', GAUGE),
```

Comment:
This might break existing user's screens, alerts, etc. Renaming a metric that's out there could be considered a breaking change.